Libraries/PeriphDrivers/Source/UART/uart_revb.h

```diff
@@ -74,14 +74,15 @@ int MXC_UART_RevB_Read(mxc_uart_revb_regs_t *uart, uint8_t *buffer, int *len);
 int MXC_UART_RevB_Write(mxc_uart_revb_regs_t *uart, const uint8_t *byte, int *len);
 unsigned int MXC_UART_RevB_ReadRXFIFO(mxc_uart_revb_regs_t *uart, unsigned char *bytes,
                                       unsigned int len);
-int MXC_UART_RevB_ReadRXFIFODMA(mxc_uart_revb_regs_t *uart, unsigned char *bytes, unsigned int len,
+int MXC_UART_RevB_ReadRXFIFODMA(mxc_uart_revb_regs_t *uart, mxc_dma_regs_t *dma,
+                                unsigned char *bytes, unsigned int len,
                                 mxc_uart_dma_complete_cb_t callback, mxc_dma_config_t config);
 unsigned int MXC_UART_RevB_GetRXFIFOAvailable(mxc_uart_revb_regs_t *uart);
 unsigned int MXC_UART_RevB_WriteTXFIFO(mxc_uart_revb_regs_t *uart, const unsigned char *bytes,
                                        unsigned int len);
-int MXC_UART_RevB_WriteTXFIFODMA(mxc_uart_revb_regs_t *uart, const unsigned char *bytes,
-                                 unsigned int len, mxc_uart_dma_complete_cb_t callback,
-                                 mxc_dma_config_t config);
+int MXC_UART_RevB_WriteTXFIFODMA(mxc_uart_revb_regs_t *uart, mxc_dma_regs_t *dma,
```

Comment:
I think you might need to add *dma parameter also to `Libraries/PeriphDrivers/Include/MAX32655/uart.h` and other ones that that use revb uart?

I'm getting an error trying to build the zephyr samples :

```
/home/rcornall/zephyrproject/modules/hal/adi/MAX/Libraries/PeriphDrivers/Source/UART/uart_revb.c:1084:13: error: too many arguments to function 'MXC_UART_ReadRXFIFODMA'
 1084 |         if (MXC_UART_ReadRXFIFODMA((mxc_uart_regs_t *)(req->uart), dma, req->rxData, req->rxLen, NULL) !=
      |             ^~~~~~~~~~~~~~~~~~~~~~
```

Reply:
Good to hear!